osu.Game/OsuGame.cs

```diff
@@ -577,6 +577,15 @@ protected override void LoadComplete()
 
             dependencies.CacheAs(idleTracker = new GameIdleTracker(6000));
 
+            var sessionIdleTracker = new GameIdleTracker(300000);
+            sessionIdleTracker.IsIdle.BindValueChanged(idle =>
+            {
+                if (idle.NewValue)
+                    SessionStatics.ResetValues();
+            });
+
+            Add(new GameIdleTracker(300000));
```

Comment:
This doesn't look right

Reply:
Nice catch.
Looks like ppy caused this in dbb8f7f.